2021 01 19 shaun liu pim (#11399)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/policyroot.md

---
title: "policyRoot resource type"

> description: "A new navigation properties binding for unifiedRoleManagementPolicy and unifiedRoleManagementPolicyAssignment to policyRoot."

Reviewer: ```suggestion
description: "Binds the unifiedRoleManagementPolicy and unifiedRoleManagementPolicyAssignment to a policyRoot."
```

author: "shauliu"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

# policyRoot resource type

> Namespace: microsoft.graph

Reviewer: Add: [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> A new navigation properties binding for unifiedRoleManagementPolicy and unifiedRoleManagementPolicyAssignment to policyRoot.

Reviewer: ```suggestion
Binds a unifiedRoleManagementPolicy and unifiedRoleManagementPolicyAssignment to a policyRoot.
```

## Methods
|Method|Return type|Description|
|:---|:---|:---|

> |[List roleManagementPolicies](../api/policyroot-list-rolemanagementpolicies.md)|[unifiedRoleManagementPolicy](../resources/unifiedrolemanagementpolicy.md) collection|Get the unifiedRoleManagementPolicy resources from the roleManagementPolicies navigation property.|

Reviewer: ```suggestion
|[List roleManagementPolicies](../api/policyroot-list-rolemanagementpolicies.md)|[unifiedRoleManagementPolicy](../resources/unifiedrolemanagementpolicy.md) collection|Get the unifiedRoleManagementPolicy resources from roleManagementPolicies.|
```

> |[List roleManagementPolicyAssignments](../api/policyroot-list-rolemanagementpolicyassignments.md)|[unifiedRoleManagementPolicyAssignment](../resources/unifiedrolemanagementpolicyassignment.md) collection|Get the unifiedRoleManagementPolicyAssignment resources from the roleManagementPolicyAssignments navigation property.|

Reviewer: ```suggestion
|[List roleManagementPolicyAssignments](../api/policyroot-list-rolemanagementpolicyassignments.md)|[unifiedRoleManagementPolicyAssignment](../resources/unifiedrolemanagementpolicyassignment.md) collection|Get the unifiedRoleManagementPolicyAssignment resources from roleManagementPolicyAssignments.|
```

<!--
## Properties

> |Property|Type|Description|

Reviewer: I don't think this table needs to be here if there are no properties.
